modules/core/src/test/java/org/apache/ignite/igfs/IgfsFragmentizerSelfTest.java

```diff
@@ -248,19 +247,16 @@ public void testDeleteFragmentizing() throws Exception {
 
         igfs.clear();
 
-        GridTestUtils.retryAssert(log, 50, 100, new CA() {
-            @Override public void apply() {
-                for (int i = 0; i < NODE_CNT; i++) {
-                    IgniteEx g = grid(i);
+        for (int i = 0; i < NODE_CNT; i++) {
+            IgniteEx g = grid(i);
 
-                    GridCacheAdapter<Object, Object> cache = ((IgniteKernal)g).internalCache(
-                        g.igfsx("igfs").configuration().getDataCacheConfiguration().getName());
+            GridCacheAdapter<Object, Object> cache = ((IgniteKernal)g).internalCache(
+                    g.igfsx("igfs").configuration().getDataCacheConfiguration().getName());
 
-                    assertTrue("Data cache is not empty [keys=" + cache.keySet() +
-                        ", node=" + g.localNode().id() + ']', cache.isEmpty());
-                }
-            }
-        });
+            boolean isCacheEmpty = GridTestUtils.waitForCondition(() -> cache.isEmpty(), 5000);
```

Comment:
In worst case we will wait for NODE_CNT * 5 sec,
but before this change w'd wait for 5 sec in total.